### PR TITLE
Add support for React / Next.JS

### PR DESCRIPTION
**Pull request by Alwinator**
I thought react support would be useful and I needed it for a project

### CODE REVIEW

**Comment by semmel** [comment]
Awesome! Thanks for the PR – I'll try to look into it in the next couple of days.

**Comment by nikwen** [comment]
What's the state of this PR?

**Comment by semmel** [comment]
@nikwen @Alwinator I did not find the time to integrate this PR. 
Reasons:
- being not my favs I don't have a setup ready for React/Next.JS to test
- outdated technique;  It really should use the [VirtualKeyboard API](https://developer.chrome.com/docs/web-platform/virtual-keyboard/) on Android.

The second point suggests that API-wise the whole project should really be a VirtualKeyboard API polyfill for Firefox and on iOS. [Apple's](https://github.com/WebKit/standards-positions/issues/16) and [Mozilla's](https://github.com/mozilla/standards-positions/issues/531) "intent to implement" the VirtualKeyboard API is right now pretty unclear. It [doesn't even appear](https://webkit.org/status/) to be "under consideration" for the WebKit team. 

So in summary: 
- I myself do not even use this project anymore. 
- And update would be a 100% rewrite.

Anyone interested in co-ownership?

**Comment by Alwinator** [comment]
Thanks for the offer, but unfortunately paid work is keeping me busy.

**Comment by nikwen** [comment]
@semmel Thank you so much for your detailed reply and the pointer to the VirtualKeyboard API!

Do you have any code that you can share where you've used the VirtualKeyboard API to achieve similar results and maybe even added workarounds for Firefox/Safari? That would significantly speed up the implementation of what you suggest.

In any case, thank you very much again for your pointer to that API.

**Comment by semmel** [comment]
@nikwen Sorry, I have not worked with the VirtualKeyboard API yet.

**Comment by nikwen** [comment]
> @nikwen Sorry, I have not worked with the VirtualKeyboard API yet.

Alright, thank you for your reply! I'm looking into the different APIs at the moment and if I end up writing any reusable code, I'll share it here.
